css-display-3/display-contents-alignment-001.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Test: flex container align-items is used for flex item regardless of intermediate display: contents ancestors</title>
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-display-3/#valdef-display-contents">
+<link rel="help" href="https://drafts.csswg.org/css-align/#align-items-property">
+<link rel="match" href="display-contents-alignment-001-ref.html">
+<style>
+  .container {
+    display: flex;
+    width: 300px;
+    height: 300px;
+    align-items: center;
+  }
+  .contents {
+    display: contents;
+    align-items: flex-start;
+  }
+  .contents > div {
+    width: 100px;
+    height: 100px;
+    background: blue;
```

Comment:
We could add `align-self: auto` here. I know it's the default but to verify that it's not doing `stretch`.